data/json/snippets/effect_on_conditions.json

```
@@ -0,0 +1,35 @@
+[
+  {
+    "type": "snippet",
+    "category": "NIGHT_MESSAGES_1",
+    "text": [
+      "You see what seems to be a new constellation in the sky.  It is quite dense and appears to be several regular polyhedrons next to each other.",
+      "You see what seems to be a new constellation in the sky.  It appears to be a spiral with the central star constantly shifting color.",
+      "You see several shooting stars.  They seem larger than normal and are all a deep purple.",
+      "You see a shooting star.  It is bright red and does not fall in a straight line.",
+      "For a brief moment you see a shape in the sky roughly the size of the moon, but square and green.",
+      "For a brief moment, the stars and moon all disappear.  When they come back, you're not sure if they're in the same places as they were."
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "NIGHT_MESSAGES_2",
+    "text": [
+      "You see what seems to be a new constellation in the sky.  It changes shape as you look from a triangle to a pentagon.",
```

```suggestion
      "You see what seems to be a new constellation in the sky.  It changes shape as you look from a triangle to a pentagon.",
      "You see what seems to be a new constellation in the sky, in the shape of a hexagon.  It's easy to pick out - there are glowing lines between the stars.",
```
Or this may be better with #3.